### ISSUE TITLE
x/vulndb: potential Go vuln in github.com/mojocn/base64Captcha: CVE-2023-45292

### ISSUE THREAD

**Issue by cangkuai** [golang/vulndb#2386]
### Acknowledgement
 * [ ]  The maintainer(s) of the affected project have already been made aware of this vulnerability.

### Description

This is a Module that generates website Captcha. I found that some special values can bypass Captcha verification.
When using store.Verify() to verify whether the Captcha is correct, if the first parameter is a non-existent id, the second parameter is an empty string, and the third parameter is true, the function will always consider the Captcha to be correct.
Sample code：
``` go
package main

import (
	"fmt"
	"strconv"
	"testing"

	"github.com/mojocn/base64Captcha"
)

func TestCat(t *testing.T) {
	var store = base64Captcha.DefaultMemStore
	fmt.Println(strconv.FormatBool(store.Verify("dsad", "", true)))
}
```
Reproduction screenshot
![Snipaste_2023-12-01_20-40-02](https://github.com/golang/vulndb/assets/33569578/a2fcd7ca-e367-4d97-93dc-7191e1d55ee8)
![Snipaste_2023-12-01_20-41-02](https://github.com/golang/vulndb/assets/33569578/183dfc4f-d84f-41ab-bbc5-6cc67711dd5a)
(I blurred my name in the picture above.)

### Affected Modules, Packages, Versions and Symbols

```shell
Module: github.com/mojocn/base64Captcha
Package: github.com/mojocn/base64Captcha
Versions:
  - Introduced: 1.3.5
Symbols:
  - store.Verify()
```

### References

https://github.com/mojocn/base64Captcha/issues/120

### Additional information

I contacted the author via email and issue, but after 6 days there was still no feedback from the author.

**Comment by cangkuai** [comment]
Thank you very much for getting the CVE number so quickly.

**Comment by cangkuai** [comment]
This vulnerability is fixed in version 1.3.6 https://github.com/mojocn/base64Captcha/commit/5ab86bd6f333aad3936f912fc52b411168dcd4a7
